### PR TITLE
Comment out monitoring cron if unused.

### PR DESCRIPTION
**Pull request by steven-basart**
Bug in cron tab setup.  Does not properly comment out the lines.

### CODE REVIEW

**Comment by steven-basart** [comment]
Gotta rebase this branch

**Review by andriy-safe-ai — CHANGES_REQUESTED** [review]

File: playbooks/roles/cron/tasks/el.yml [hunk 1]
```diff
@@ -35,15 +35,15 @@
 - name: Create a commented Slurm monitoring cron file under /etc/cron.d
   cron:
     name: slurm monitoring
-    minute: "*"
+    minute: "#*"
```
[the remark this answers is not in the document]
Comment (steven-basart) [reply playbooks/roles/cron/tasks/el.yml]: Good find! I like it.

I switched to using the built in ansible disabled.

File: playbooks/roles/cron/tasks/el.yml [hunk 2]
```diff
@@ -35,15 +35,15 @@
...
     user: '{{ ansible_user }}'
     job: "#source /opt/oci-hpc/monitoring/env; /opt/oci-hpc/monitoring/monitor_slurm.sh >> /opt/oci-hpc/monitoring/monitor_slurm_`date '+\\%Y\\%m\\%d'`.log 2>&1"
```
[the remark this answers is not in the document]
Comment (steven-basart) [reply playbooks/roles/cron/tasks/el.yml]: Switched everything to `disabled: true`

File: playbooks/roles/cron/tasks/el.yml [hunk 3]
```diff
@@ -35,15 +35,15 @@
...
   when: not autoscaling_monitoring | bool
 
 - name: Create a commented OCI monitoring cron file under /etc/cron.d
   cron:
     name: OCI monitoring
-    minute: "*"
+    minute: "#*"
     user: '{{ ansible_user }}'
     job: "#source /opt/oci-hpc/monitoring/env; /opt/oci-hpc/monitoring/monitor_oci.sh >> /opt/oci-hpc/monitoring/monitor_oci_`date '+\\%Y\\%m\\%d'`.log 2>&1"
```
[the remark this answers is not in the document]
Comment (steven-basart) [reply playbooks/roles/cron/tasks/el.yml]: Twitter needs the hashtags bro!

**Comment by steven-basart** [comment]
fixed it @andriy-safe-ai

**Review by andriy-safe-ai — APPROVED** [review]
LGTM

**Pull request merged**